client/zecale/cli/zecale_nested_verification_key_hash.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2015-2020 Clearmatics Technologies Ltd
+#
+# SPDX-License-Identifier: LGPL-3.0+
+
+from .command_context import CommandContext
+from .utils import load_verification_key
+from zeth.core.utils import hex_to_uint256_list
+from click import command, argument, Context, pass_context
+
+
+@command()
+@argument("verification_key_file")
+@pass_context
+def nested_verification_key_hash(
+        ctx: Context,
+        verification_key_file: str) -> None:
+    """
+    Query the aggregator_server for the hash of a specific nested verification
+    key. The verification key does not have to be registered.
+    """
+    cmd_ctx: CommandContext = ctx.obj
+
+    # Load key and call the GetNestedVerificationKeyHash method
+    vk = load_verification_key(cmd_ctx.zksnark, verification_key_file)
+    aggregator_client = cmd_ctx.get_aggregator_client()
+    vk_hash_hex = aggregator_client.get_nested_verification_key_hash(vk)
```

Comment:
I did it like this because, as far as I could tell, the `r1cs_gg_ppzksnark_verification_key_variable` does not pad the bit-string to the nearest byte (or other word-size) boundary, so current it's actually a little difficult to replicate the computation on the client.

Options are obvious, to could change the circuit to pad each element.  Or we could use the pairing parameters (from a later zeth PR) to actually do the bit-shift on the client, which would be pretty complex.  This seems like the easiest approach for now.

Reply:
I see, thanks. Fine for now then :)